benchmarks/collections.js

```
@@ -3,90 +3,113 @@ description: Set and Map collection benchmarks
   });
 });
 
 suite("Map operations", () => {
-  bench("set 50 entries", () => {
-    const m = new Map();
-    Array.from({ length: 50 }, (_, i) => i).forEach((i) => m.set("key" + i, i));
+  bench("set 50 entries", {
+    setup: () => Array.from({ length: 50 }, (_, i) => i),
+    run: (items) => {
+      const m = new Map();
+      items.forEach((i) => m.set("key" + i, i));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)